.pylintrc

```diff
@@ -52,7 +52,7 @@ ignore=CVS
 # ignore-list. The regex matches against paths and can be in Posix or Windows
 # format. Because '\\' represents the directory delimiter on Windows systems,
 # it can't be used as an escape character.
-ignore-paths=
+ignore-paths=.*/migrations
```

Comment:
Why do you want to ignore them in every/arbitrary directory? Be more precise when disabling useful checks.

Reply:
I added blog.

Reply:
Marking as resolved.